ouroboros-consensus-cardano-test/src/Test/Consensus/Cardano/Examples.hs

```diff
@@ -121,6 +143,7 @@ instance Inject Examples where
       , exampleQuery            = inj (Proxy @(SomeSecond BlockQuery)) exampleQuery
       , exampleResult           = inj (Proxy @SomeResult)              exampleResult
       , exampleAnnTip           = inj (Proxy @AnnTip)                  exampleAnnTip
+      , exampleLedgerConfig     = mempty -- We rely on `combineEras` to make use of the example ledger configs
```

Comment:
My initial expectation is that this field should be a use of `inj` just like all the others. I can imagine that it was not obvious how to define the different examples on a per-era basis for Shelley, though. I have a couple ideas for that, if you confirm that's the only reason you went with `mempty` for now.

Reply:
Yes, indeed we're trying to inject the ledger config for a single era but we need a ledger config for all eras to produce an example here. Happy to hear your ideas.

Reply:
So I think I've found a simpler solution. I think we were implicitly insisting on keeping the `Inject Examples` instance, when really we can just replace it's use with a `mkCardanoExamples :: NP Examples (CardanoEras Crypto) -> Examples (CardanoBlock Crypto)` which is able to use `inject` as usual for other fields but also combine the per-era ledger configs into a HFC ledger config. This solution feels correct and like a simplification to the PR.